fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordStoreState.java

```diff
@@ -314,13 +347,16 @@ public RecordStoreState withIndexesInState(@Nonnull final List<String> indexName
     /**
      * Create a new version of this {@code RecordStoreState}, but with additional {@link IndexState#WRITE_ONLY} indexes.
      *
-     * @param writeOnlyIndexNames the indexes to be marked as WRITE_ONLY. If the index is already DISABLED, it will
-     * stay disabled, but will otherwise be set to WRITE_ONLY.
+     * <p>
+     *  If an index is already {@code DISABLED}, it will stay disabled, but will otherwise be set to WRITE_ONLY.
+     * </p>
+     *
+     * @param writeOnlyIndexNames the indexes to be marked as WRITE_ONLY.
```

Comment:
As long as you are editing comments, I think we are not using a period at the end of `@param` and `@return` descriptions.